src/page-components/unicorns/profile-header/profile-header.module.scss

```diff
@@ -110,7 +110,8 @@
 	}
 
 	.socialText {
-		margin-left: 16px;
+		margin-left: 0.5em;
+		padding: 0.2em 0;
```

Comment:
Instead of doing this, let's set:

```css
.socialBtnLink > a {
    align-items: center;
}
```

Which also solves the problem, but doesn't rely on hardcoding padding values